tmt-app/src/Components/QuestionCard.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import "../Styling/Home.css";
+import { Typography, Grid, Box } from "@mui/material";
+import { CustomTheme } from "../Styling/CustomStyling";
+
+const QuestionCard = ({ item }) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const [isSelected, setIsSelected] = useState(false);
+
+  const handleHover = () => {
+    setIsHovered(true);
+  };
+
+  const handleLeave = () => {
+    setIsHovered(false);
+  };
+
+  const handleClick = () => {
+    setIsSelected(!isSelected);
+  };
+
+  const style = {
+    margin: "0.6rem",
+    padding: "1rem",
+    gap: "0.5rem",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+    width: "240px",
+    minHeight: "152px",
+    boxShadow: "6px 6px 16px rgba(0, 0, 0, 0.16)",
+    borderRadius: "36px",
+    flex: "none",
+    flexGrow: 0,
+    textAlign: "center",
+    transition: "all 0.3s ease-in-out",
+    border: isSelected
+      ? "none"
+      : isHovered
+      ? `12px solid ${CustomTheme.palette.RedBean.main}`
+      : "none",
+    background: isSelected
+      ? CustomTheme.palette.ThaiTea.main
+      : isHovered
+      ? "white"
+      : CustomTheme.palette.BobaHighlight.main,
+    color: isHovered ? "black" : "white",
+  };
+
+  return (
+    <Grid
+      item
+      key={item.title}
+      style={style}
```

Comment:
we should consider using the `sx` prop over `style` when possible https://mui.com/system/getting-started/the-sx-prop/